java/cdk/src/main/java/sleeper/cdk/Utils.java

```diff
@@ -110,12 +112,22 @@ public static String truncateTo64Characters(String input) {
         return truncateToMaxSize(input, 64);
     }
 
+    private static final int AVAIL_RESOURCE_NAME_LENGTH = 64 - ("sleeper-".length() + ID_MAX_LENGTH + 1);
```

Comment:
To match the naming scheme of `ID_MAX_LENGTH` maybe this could be named `RESOURCE_NAME_MAX_LENGTH`?

Reply:
I just got rid of this since it makes it slightly harder to change resource names in the future.